site/assignments/tic-tac-toe/tic-tac-toe.rst

```diff
@@ -244,6 +244,157 @@ Render Board
+final result. Fortunately, with all the core functionality already written, much of this is just a matter of putting
+things together.
+
+The setup is fairly straight forward:
+
+    * Prompt the user for the game size
+    * Create the game board with the specified size
+    * Setup some bookkeeping variables
+
+        * Move counters
+        * Current player symbol
+        * A flag for if the game is over
+
+
+The game loop is going to do much of the work. It needs to:
+
+    * Run while no one has won yet and there are still valid moves left
+    * Set the current player symbol
+    * Render and display the board
+    * Display the current move counter
+    * Prompt the user for a move until they provide a valid move
+    * Apply the move to the board
+    * Increase the move counter
+    * Check for a winner
+
+
+Once the game ends, final details need to be displayed to the players. This will include:
+
+    * The final game board
+    * Say who won the game and in how many moves *or* state that it's a cat's game (which means no one won)
+
+
+Some additional things to note about Xtreme tic-tac-toe:
+
+   * X always goes first
+   * The game can end in a draw if there are no more valid moves available (this is called a "cat's game")
+   * X will always win a game that's smaller than 3x3 (think about why that is)
+
+
+Below is some pseudocode that you may find helpful. For the most part, it is just restating the above points. The first
+line of code, the ``if`` statement, is not actual pseudocode and is something you need in your code. It is required for
+our marking and basically means that the code within the block will only run if we are directly running this script. For
+example, if one were to ``import`` your code into another script (which is done for marking), Python would try to run
+all the code within the imported script. By having this line of code, it says to not bother running the block unless the
+script was ran directly.
+
+.. code-block:: python
+
+    # Not actual pseudocode --- makes it so the import
+    # from the unit tests do not break things
+    if __name__ == "__main__":
+        # Setup code
+        Get the game size
+        Create a game board of the size
+        Initialize a move counter
+        Set current player symbol
+        Set game over flag to false
+
+        # Game loop
+        while the game is not over
+            Set current player symbol
+            Render and display the game board and move counter
+            Read input from the user until valid input is entered
+            Apply the move to the game board
+            Increment move counter
+            Check if player has won
+
+        # Game ending part
+        Render and display the game board
+        Print out which player won and in how many moves or if no one won
+
+
+Below is an example of a full game with player ``"X"`` winning. Notice that player ``"X"`` entered an invalid move for
+their first move.
+
+.. code-block:: python
```

Comment:
This block is `text`, not python.

Sort of, same for the last block, but it also contains some real python so /shrug.